packages/eas-cli/src/project/publish.ts

```diff
@@ -0,0 +1,308 @@
+import { Platform } from '@expo/config';
+import crypto from 'crypto';
+import fs from 'fs';
+import { uniqBy } from 'lodash';
+import path from 'path';
+
+import { PartialManifestAsset } from '../graphql/generated';
+import { PublishMutation } from '../graphql/mutations/PublishMutation';
+import { PublishQuery } from '../graphql/queries/PublishQuery';
+import { PresignedPost, uploadWithPresignedPostAsync } from '../uploads';
+
+export const TIMEOUT_LIMIT = 60_000; // 1 minute
+let STORAGE_BUCKET: string;
+export const Platforms: PublishPlatforms[] = ['android', 'ios']; // TODO-JJ allow users to specify this in app.js
```

Comment:
There is different capitalization between `AppPlatform` and `import { Platform } from '@expo/config';`.

On the backend we've been using the lowercase typing supplied by the latter.

It seems like these should be unified, but that is out of the scope of this PR. For now to keep consistency with the backend will prefer the lowercase type.

Reply:
Looks like these are used inconsistently between different parts of the API as well – compare `Build`, which uses the `AppPlatform` enum (`IOS`, `ANDROID`):
https://github.com/expo/eas-cli/blob/58919d37ab3b42585b12ac593159f5e004bd2d6b/packages/eas-cli/src/graphql/generated.ts#L695
and `Update`, which uses string (expecting lowercase `'ios'` or `'android'` I guess):
https://github.com/expo/eas-cli/blob/58919d37ab3b42585b12ac593159f5e004bd2d6b/packages/eas-cli/src/graphql/generated.ts#L971
There should prefereably be only one way to represent platforms in the GraphQL API, and I think the enum is better because it's less prone to error (no confusion between `'iOS'`, `'ios'` or `'IOS'`, values validated by the schema). But you are right that unifying this doesn't belong in this PR.